portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+/** A user comment. */
+public final class Comment {
+
+    private final long id;
+    private final String comment;
```

Comment:
I do see "Your Name" is required in making comments, but not seeing it in the data schema? Are you considering having it here or removing it from form?

Reply:
i use the name and combine it with the comment to generate a user comment:
ie usercomment = name + "says" + comment